source/PlatformerHero.hx

```diff
@@ -0,0 +1,63 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+
+/**
+ * ...
+ * @author Cory Jackson
+ */
+class PlatformerHero extends FlxSprite 
+{
+	var colorArray = [FlxColor.RED, FlxColor.YELLOW];
+	var i = 0;
+	
+	public function new(?X:Float=0, ?Y:Float=0, ?SimpleGraphic:FlxGraphicAsset) 
+	{
+		super(X, Y, SimpleGraphic);
+		this.makeGraphic(25, 150, FlxColor.YELLOW);
+		this.acceleration.y = 60;
+		this.maxVelocity.y = 150;
+	}
+	
+	override public function update(elapsed:Float):Void
+	{
+		var leftPressed:Bool = FlxG.keys.pressed.LEFT;
+		var rightPressed:Bool = FlxG.keys.pressed.RIGHT;
+		var upPressed:Bool = FlxG.keys.pressed.Z;
+		var extraPressed:Bool = FlxG.keys.justPressed.V;
+
+		if (leftPressed) { //left
+			if (leftPressed && rightPressed){ //not null movement, sad days
+					leftPressed = rightPressed = false;
+			}
+			else {
+			this.acceleration.x = -50;
+			}
+		} 
+		else if (rightPressed) { //right
+			this.acceleration.x = 50;
+		}
+		else if (upPressed){ //up
+			this.velocity.y = -200;
+		}
+		else { //stop
+			this.acceleration.x = 0;
+			this.acceleration.y = 60;
```

Comment:
I don't think acceleration.y actually needs to be set to 60 here, since it never changes in any of your input code.